.github/dependabot.yml

```diff
@@ -4,23 +4,26 @@ updates:
     directory: "/"
     schedule:
       interval: daily
-    open-pull-requests-limit: 10
+    # dependabot does not work with yarn v2, disabling for now
+    open-pull-requests-limit: 0
     commit-message:
       prefix: chore
       include: scope
   - package-ecosystem: npm
     directory: "/packages/tokens"
     schedule:
       interval: daily
-    open-pull-requests-limit: 10
+    # dependabot does not work with yarn v2, disabling for now
+    open-pull-requests-limit: 0
     commit-message:
       prefix: chore
       include: scope
   - package-ecosystem: npm
     directory: "/packages/components"
     schedule:
       interval: daily
-    open-pull-requests-limit: 10
+    # dependabot does not work with yarn v2, disabling for now
+    open-pull-requests-limit: 0
```

Comment:
Should we delete this file and not even run the job?

Reply:
I had hope that one day dependabot would fix this issue and then we could edit the config to work better :')